### PR TITLE
handle ID collisions for multiple locations on a single place

### PR DESCRIPTION
**Pull request by paregorios**
Addresses https://github.com/isawnyu/pleiades-gazetteer/issues/457

### CODE REVIEW

**Comment by paregorios** [comment]
Script has been tested locally and on production.

**Comment by paregorios** [comment]
Test data:

[units-blocks-streets-2.json.zip](https://github.com/isawnyu/pleiades3-buildout/files/7469211/units-blocks-streets-2.json.zip)

Run like:

```bash
/srv/python27-apps/pleiades4$ bin/instance1 run scripts/place_maker.py --message='imported with place_maker.py by Tom Elliott' --owner=achen --groups=ydea --creators=achen --contributors=kcl,thomase,jbecker --tags='YDEA project' --dry-run /home/thomase/units-blocks-streets-2.json
```

File: scripts/place_maker.py [hunk 1]
```diff
@@ -133,14 +148,26 @@ def set_tags(content, args):
...
-    parser.add_argument('--creators', nargs='+', default=[],
-                        dest='creators', help='Creators. Separated by spaces.')
-    parser.add_argument('--contributors', default=[],
-                        dest='contributors', nargs='+', help='Contributors. Separated by spaces.')
-    parser.add_argument('--tags', default=[], dest='subjects', nargs='+',
-                        help='Tags (subjects). Separated by spaces.')
+    parser.add_argument('--groups',
+                        default=[],
+                        dest='groups', 
+                        type=lambda s: re.split(r' |,\s*', s),
```
Comment (alecpm) [inline scripts/place_maker.py:154]: This is a fairly unusual use of `type`, but I guess it works as a means to coerce a string into a list with more complex formatting than would be provided by `nargs='+'`.

**Review by alecpm — APPROVED** [review]
This looks OK to me.

**Pull request merged**